docs/provision-in-gcr.md

```diff
@@ -0,0 +1,86 @@
+
+As Amazon Web Services China operates seperately from Amazon Web Services Global regions, due to feature availability and regulatory requirements, the SaaS Boost provision experience in GCR is unique. This document provides guidance of provisioning AWS SaaS Boost in GCR.
+
+## Feature Availability and Implementation Differences
+1. System User Service is different in the following ways:
+    - Amazon Cognito User Pools are not currently available in the GCR Regions.
+    - Keycloak(Open Source Identity and Access Management) is provided as another identity provider.
+2. Admin Web UI is different in the following ways:
+    - You can’t use the default CloudFront domain, `*.cloudfront.cn`, to serve content. You must add an alternate domain name, also known as a CNAME, to your CloudFront distributions, and then use that domain name in the URLs for your content. You also must have [an ICP registration](https://www.amazonaws.cn/en/about-aws/china/#ICP_in_China). In addition, just as with the global CloudFront service, to serve content over HTTPS, you must use an SSL/TLS certificate with your alternate domain name.
+    - Amazon CloudFront in the China Regions currently does not support Amazon Certificate Manager. You must get an SSL/TLS certificate from a different third-party certificate authority (CA) and then upload it to the IAM certificate store.
+3. Amazon SES is unique in the following ways:
+    - Amazon SES is not availible in GCR regions.
+4. Amazon QuickSight is unique in the following ways:
+    - Amazon QuickSight is not availible in GCR regions.
+
+## Provision AWS SaaS Boost into your AWS GCR Account
+To prepare the installation process, perform the following steps as needed:
+1. You have a domain and your domain is [ICP registered](https://www.amazonaws.cn/en/about-aws/china/#ICP_in_China).
+2. You need a root domain name and individual domain names for your keycloak installation and the admin web UI. As part of this you need one certificate in ACM for keycloak domain, and one certificate in IAM that covers admin web UI domain. Either wildcard certificates (e.g. *.example.com) or specific certificates (e.g. keycloak.example.com). 
+    - You need to create a public hosted zone in AWS Route53 for your domain.
+    - Request or upload [certificate to ACM](https://docs.aws.amazon.com/acm/latest/userguide/gs-acm-request-public.html)
+    - Upload [certificate to IAM](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/cnames-and-https-procedures.html). 
+    - There are many free and paid options you can use to generate a SSL/TLS certificate. The example we use here is [letsencrypt.org](https://letsencrypt.org/) for a free SSL/TLS certificate.
```

Comment:
Actually @goodbyedavid after some more conversation in the team we've decided it's best if we leave this out completely. It's safer for the project from a liability perspective if we allow customers to discover their own mechanism for generating SSL/TLS certificates. Please remove this, and we can merge this PR.

Reply:
@PoeppingT This part is now removed.